test/test_ops.py

```diff
@@ -1796,11 +1800,34 @@ def test_refs_are_in_decomp_table(self, op):
     "fft.rfftn",
     "svd",
     "linalg.svd",
-)
+]
 
+backward_skips = fake_striding_skips + [
+    "linalg.cond",
+    "linalg.matrix_norm",
+    "linalg.norm",
+    "linalg.svd",
+    "linalg.svdvals",
+    "nn.functional.binary_cross_entropy_with_logits",
+    "nn.functional.huber_loss",
+    "nn.functional.logsigmoid",
+    "nn.functional.multilabel_soft_margin_loss",
+    "pca_lowrank",
+    "roll",
+    "svd_lowrank",
+    "sgn",
+]
```

Comment:
Are these actually permanent skips?  A lot of these look like they could be xfails

Reply:
they are xfails actually